### PR TITLE
use zstandard library

### PR DESCRIPTION
**Pull request by sshane**
seems to be a much more maintained library

edit: qlogs compress slightly faster but rlogs take 2x longer!

### CODE REVIEW

**Comment by maxime-desroches** [comment]
Yes, and they also have pre-built python3.12 wheels!

**Comment by andiradulescu** [comment]
@sshane can you try to reduce the compression level step by step and see when rlogs don’t take 2x anymore? I think it uses a newer version of zstd. I only used zstandard with the default level.

**Comment by sshane** [comment]
I'm confused, the `zstd` library is so much faster than the other two with the same zstd version (1.5.5), will look into it.

```
comma@comma-1a8caafe:/data/openpilot$ python test.py 
5.219517157995142  # zstd
9.818812099983916  # zstandard
10.366369541501626  # pyzstd
```

**Comment by sshane** [comment]
I assumed that a default argument value of `threads=0` meant single threaded, but apparently not :upside_down_face: https://github.com/sergey-dryabzhinsky/python-zstd/blob/eba9e633e0bc0e9c9762c985d0433e08405fd097/src/python-zstd.c#L93

File: selfdrive/test/test_onroad.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 import subprocess
 import time
 import numpy as np
-import zstd
+import zstandard as zstd
```
Comment (sshane) [inline selfdrive/test/test_onroad.py:11]: Says you can create a zstd context to compress in tight loops, but I didn't notice a huge difference in our case.

**Pull request merged**